frontend/src/components/MunchGrid.tsx

```diff
@@ -0,0 +1,99 @@
+import Paper from '@mui/material/Paper';
+import React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+
+export interface MunchCardItem {
+    item_name: string,
+    restaurant_name: string,
+    price: number,
+    description: string,
+}
+export interface MunchGridProps {
+    cards: Array<MunchCardItem>,
+    show: string
+}
+
+function MunchGrid(props: MunchGridProps): React.ReactElement {
+    const [order, setOrder] = React.useState<Order>('asc');
+    const [orderBy, setOrderBy] = React.useState<keyof MunchCardItem>('item_name');
+    const [page, setPage] = React.useState(0);
+    const [cardsPerPage, setCardsPerPage] = React.useState(10);
+    const [cardsPerRow, setCardsPerRow] = React.useState(3);
+
+    return (
+        <Paper sx={{display:props.show}} >
+            <Box sx={{ flexGrow: 1}}>
+            {stableSort(props.cards, getComparator(order, orderBy)).slice(page * cardsPerPage, page * cardsPerPage + cardsPerPage).map((card, index) => {
+              return (
+                <Card sx={{ maxWidth: 345 }}>
+                  <CardMedia
+                    sx={{ height: 140 }}
+                    image="/static/images/cards/contemplative-reptile.jpg"
+                    title="{card.item_name}"
+                  />
+                <CardContent>
+                  
+                  <Typography gutterBottom variant="h5" component="div">
+                    {card.item_name}
+                  </Typography>
+                  <Typography gutterBottom variant="h5" component="div">
+                    {card.price}
+                  </Typography>
+                  <Typography gutterBottom variant="h5" component="div">
+                    {card.restaurant_name}
+                  </Typography>
+                  <Typography variant="body2" color="text.secondary">
+                    {card.description}
+                  </Typography>
+
+                </CardContent>
+              </Card>
+              );
+            })}
+            </Box>
+        </Paper>
```

Comment:
Is card dummy data a part of this task/PR or should this section be removed until later tasking is worked? 
```suggestion
        <Paper sx={{display:props.show}} >
            <Box sx={{ flexGrow: 1}}>
            </Box>
        </Paper>
```

Reply:
removed. the data is what is displayed on the table, by the way. but I removed it in this branch and added it to a clone for later.